src/jit/codegenarm64.cpp

```diff
@@ -5778,7 +5777,8 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
                                                // as that is how much stack is allocated for this LclVar
                 isHfa          = varDsc->lvIsHfa();
                 gcPtrCount     = varDsc->lvStructGcCount;
-                structGcLayout = varDsc->lvGcLayout;
+                for(unsigned i = 0; i < gcPtrCount; ++i)
+                  gcPtrs[i] = varDsc->lvGcLayout[i];
```

Comment:
It looks like the job is unhappy with more than that. The output is below, with how each line was before the change and what it wants it to change to.

<pre>
09:12:53 At Line 5760 Before:
09:12:53             BYTE     gcPtrs[MAX_ARG_REG_COUNT] = {};         // TYPE_GC_NONE = 0
09:12:53 After:
09:12:53             BYTE     gcPtrs[MAX_ARG_REG_COUNT] = {}; // TYPE_GC_NONE = 0
09:12:53 At Line 5761 Before:
09:12:53             unsigned gcPtrCount;                             // The count of GC pointers in the struct
09:12:53 After:
09:12:53             unsigned gcPtrCount;                     // The count of GC pointers in the struct
09:12:53 At Line 5778 Before:
09:12:53                 isHfa          = varDsc->lvIsHfa();
09:12:53 After:
09:12:53                 isHfa      = varDsc->lvIsHfa();
09:12:53 At Line 5779 Before:
09:12:53                 gcPtrCount     = varDsc->lvStructGcCount;
09:12:53 After:
09:12:53                 gcPtrCount = varDsc->lvStructGcCount;
09:12:53 At Line 5780 Before:
09:12:53                 for(unsigned i = 0; i < gcPtrCount; ++i)
09:12:53 After:
09:12:53                 for (unsigned i = 0; i < gcPtrCount; ++i)
09:12:53 At Line 5780 Before:
09:12:53                 for (unsigned i = 0; i < gcPtrCount; ++i)
09:12:53                   gcPtrs[i] = varDsc->lvGcLayout[i];
09:12:53 After:
09:12:53                 for (unsigned i = 0; i < gcPtrCount; ++i)
09:12:53                     gcPtrs[i] = varDsc->lvGcLayout[i];
09:12:53 At Line 5781 Before:
09:12:53                     gcPtrs[i] = varDsc->lvGcLayout[i];
09:12:53 After:
09:12:53                     gcPtrs[i]   = varDsc->lvGcLayout[i];
</pre>